addons/mail_group/models/mail_group.py

```diff
@@ -234,6 +235,10 @@ def _alias_get_creation_values(self):
         values['alias_defaults'] = literal_eval(self.alias_defaults or '{}')
         return values
 
+    def action_switch_close(self):
```

Comment:
Would prefer having a method / action, it avoids issues when doing multi click, state inconsistency, ... (aka: action_close and action_open).